app/Resources/views/base.html.twig

```diff
@@ -1,13 +1,31 @@
+{% import "::macros.html.twig" as macros %}
 <!DOCTYPE html>
-<html>
-    <head>
-        <meta charset="UTF-8" />
-        <title>{% block title %}Welcome!{% endblock %}</title>
-        {% block stylesheets %}{% endblock %}
-        <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
-    </head>
-    <body>
-        {% block body %}{% endblock %}
-        {% block javascripts %}{% endblock %}
-    </body>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="/favicon.ico" rel="shortcut icon" type="image/x-icon"/>
+    <title>{% if title is defined %}{{ title }} - {% endif %}PhpOfBy</title>
+    <link rel="stylesheet" href="{{ asset('css/common.css') }}"/>
+    {% block stylesheets %}{% endblock %}
+    {{ macros.getGoogleAnalyticsCode(google_analytics_id) }}
```

Comment:
@chilic Google recommends to add code before the closing `</head>` tag.
https://support.google.com/analytics/answer/1008080?hl=en-GB (open `Static website` or `Dynamic website` section ).